### ISSUE TITLE
Media Library: Atomic sites unable to upload certain file types

### ISSUE THREAD

**Issue by samiff** [Automattic/wp-calypso#30251]
#### Steps to reproduce
1. On an AT site, try to upload a `.docx` file to the Media Library via Calypso.
[test-word.docx](https://github.com/Automattic/wp-calypso/files/2775071/test-word.docx)

#### What I expected
* The file uploads without error.

#### What happened instead
* Error displayed `1 file could not be uploaded because an error occurred while uploading.`
* The same file uploads fine in wp-admin on AT sites.
* The same file uploads fine on simple sites via Calypso & wp-admin.

#### Context / Source
#user-report 6276754-hc

**Comment by tanjoymor** [comment]
I've experienced this exact same described issue but with a PPTX file. 

2072461-hc
1732445-zen

**Comment by arunsathiya** [comment]
8606857-hc

**Comment by gpmw** [comment]
This issue has come around again here:
1732445-zen

Checking for updates.

**Comment by susanjanec** [comment]
This issue came up with user trying to upload a PPT file via Calypso when editing a page and trying to add in Media. They received the following error - `1 file could not be uploaded because an error occurred while uploading`

When the user uploaded to Media via WP-Admin they did not get an error.

HC 11590774-hc and follow up is here 1910917-zen

**Comment by lancewillett** [comment]
This is a "gap" in functionality between Simple and Atomic, and we should track it in our list of parity issues.

**Comment by lancewillett** [comment]
This needs re-testing with a variety of file types. I tried uploading this `.docx` Microsoft Word document
[20182005   Prompt 12.docx](https://github.com/Automattic/wp-calypso/files/4542575/20182005.Prompt.12.docx) and it worked as expected.

To test:

1. Powerpoint — `.pptx`
2. Word  — `.docx` (various)
3. Excel — `.xlsx`

Then, when new reports come in, it's possible the format is supported — but the file in question has an issue with it.

> Could an issue uploading an .srt file be related to this as well?

No, I don't think so. That file type shouldn't be supported, right? `SubRip` — maybe suggest plain text `.txt` or hosting it on another service and linking to it.

**Comment by lancewillett** [comment]
Testing on Atomic site today, all 3 Microsoft Office documents I tried worked as expected.

Steps to repeat:

1. Create new post from WP Admin > Posts > Add New
2. Upload files via `File` block
3. Publish

<img width="843" alt="Screen Shot 2020-04-28 at 10 54 12" src="https://user-images.githubusercontent.com/66797/80520652-9a6c7c00-893e-11ea-8b96-b08f56c0739f.png">

**Comment by lancewillett** [comment]
Next, I tested on the same Atomic site from two other places:

1. Calypso Media Library: `https://wordpress.com/media/example.com`
2. WP Admin > Media Library: `/wp-admin/upload.php`

Both cases all 3 files uploaded correctly.

Closing this as "works for me."

If you run into new issues reported by customers, I suggest posting on Atomic Requests P2 to troubleshoot. Looking at the file MIME type, file content, and maybe settings for the individual sites in case a plugin or something else is blocking it.
